src/pageObject.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
-class PageObject {
+class PageObject extends Header {
```

Comment:
PageObject should not extend Header. Instead, it should have a header property that is an instance of the Header class. 🚩

Reply:
Done